src/audio/google/google_rtc_audio_processing.c

```diff
@@ -556,22 +595,14 @@ static int google_rtc_audio_processing_free(struct processing_module *mod)
 
 	GoogleRtcAudioProcessingFree(cd->state);
 	cd->state = NULL;
-#if !CONFIG_IPC_MAJOR_4
-	rfree(cd->output_buffer);
-#endif
-	rfree(cd->aec_reference_buffer);
 	GoogleRtcAudioProcessingDetachMemoryBuffer();
-	rfree(cd->memory_buffer);
-#if CONFIG_IPC_MAJOR_4
-		rfree(cd->process_buffer);
-#else
-		rfree(cd->raw_mic_buffer);
-#endif
 	comp_data_blob_handler_free(cd->tuning_handler);
 	rfree(cd);
 	return 0;
 }
 
+#define PIPE_ID(srcsink) ((srcsink)->audio_stream_params->pipeline_id)
```

Comment:
I wrote also a note in other PR. 
use 
```
static inline uint32_t sink_get_pipleine_id(struct sof_sink *sink)
{
	return sink->audio_stream_params->pipeline_id;
}
```

and put it in sink_api.h

I would like to keep as many sink/source internals  "private" as possible